### PR TITLE
Add week selection

### PR DESCRIPTION
**Pull request by GamerBene19**

### CODE REVIEW

File: src/components/navigation/weekday-selection/WeekdaySelection.tsx [hunk 1]
```diff
@@ -1,19 +1,71 @@
...
+        undefined,
+        { shallow: true },
+      );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dateChangedManually, selectedDate]);
+
+  // Computes the date that is set by the `useEffect` below
+  const determineDateToSet = useCallback(
+    (dateURL: NextRouter['query'][string], afterFriday: boolean) => {
+      const dateURLParsed =
```
Comment (stormofice) [inline src/components/navigation/weekday-selection/WeekdaySelection.tsx]:
```suggestion
    (dateUrl: NextRouter['query'][string], afterFriday: boolean) => {
      const dateUrlParsed =
```

File: src/util/dayjs.ts [hunk 2]
```diff
@@ -15,9 +15,7 @@ export const currentDate = dayjs();
 export const afterFriday = dayjs().weekday() > 4;
 // Date of monday of the week that will be displayed when page is loaded initially
 // If it's after friday this is the next monday
```
Comment (stormofice) [inline src/util/dayjs.ts]: Is this comment still relevant?

**Review by stormofice — COMMENTED** [review]
Liking the changes overall, just got some small questions.

File: src/components/navigation/weekday-selection/WeekdaySelection.tsx (hunk 1 above)
Comment (GamerBene19) [reply src/components/navigation/weekday-selection/WeekdaySelection.tsx]: Done.

File: src/util/dayjs.ts (hunk 2 above)
Comment (GamerBene19) [reply src/util/dayjs.ts]: Changed/Rephrased it

**Review by stormofice — APPROVED** [review]

**Pull request merged**